tests/functional/utilities/reportHandler.js

```diff
@@ -88,7 +89,7 @@ describe('reportHandler::_crrRequest', function testSuite() {
             backbeatMetrics = new backbeat.Metrics({
                 redisConfig: config.redis,
                 validSites: sites,
-                internalStart: Date.now() - 900000,
+                internalStart: Date.now() - (EXPIRY * 1000),
```

Comment:
quick comment: why do you use a variable `EXPIRY` here when you simply used the number in the function file?

Reply:
ah, let me add an expiry constant to the function file. I think it will be nice to have it more explicit if changes in the future will need to be made here.

I think this could use a refactor to make more manageable in the future though, but for now, just want to increase expiry

Reply:
I had guessed that adding the variable here was so that it would be more manageable for future changes :) I agree with having the variable in both files

Reply:
I'd say for purposes of maintainability, it would be ideal to have the constant variable the same across files (e.g., instead of using both `EXPIRY` and `CRR_METRIC_EXPIRY`).